src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +48,9 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream()
+                .filter(e -> e > 3)
+                .map(e -> e * 2)
+                .reduce(0, Integer::sum);
```

Comment:
```suggestion
                .filter(number -> number > 3)
                .map(number -> number * 2)
                .reduce(0, Integer::sum);
```

람다 표현식에서도 축약된 변수명보다는 정확한 변수명을 사용하는 것이 좋을 것 같습니다.